src/shared/validate/validate.ts

```diff
@@ -140,7 +140,7 @@ export async function validateW3C<T extends OpenAttestationDocument>(
     throw new Error("Property 'type' must exist and be an array");
   }
   if (!credential.type.includes("VerifiableCredential")) {
-    throw new Error("Property 'type' must have VerifiableCredential as one of the items");
+    throw new Error("Property 'type' must have OpenAttestationVerifiableCredential as one of the items");
```

Comment:
I think the error message might have been accidentally changed? Since we're checking if `type` contains `VerifiableCredential`.

Reply:
intellij :see_no_evil: good catch =)